app/resources/documentation/manual/index.md

```diff
@@ -440,11 +440,13 @@ TrenchBroom gives you various options on how to create new objects. In the follo
 
 ### Creating Simple Shapes
 
-The easiest way to create a new brush is to just draw it with the mouse. To draw a brush, make sure nothing is currently selected and left drag in the 3D viewport or any of the 2D viewports. If you draw a brush in the 3D viewport, its shape is controlled by the point where you initially started your drag, the point currently under the mouse, and the current grid size. Essentially, you will draw your brush on the XY plane, and its height will be set to the current grid size. But it is possible to change the height while drawing a brush by holding #key(Alt). Whenever #key(Alt) is held, you can change the brush height, otherwise you control the X/Y extents of the brush.
+The easiest way to create a new brush is to just draw it out with the mouse using the Simple Shape Tool which is enabled by default when nothing else is selected or any other tool active. Left drag in the 3D viewport or any of the 2D viewports.
```

Comment:
I'd like to suggest splitting the first sentence into two: 

> The easiest way to create a new brush is to just draw it out with the mouse using the Simple Shape Tool. This tool is enabled by default when nothing else is selected and no other tool is active.

Reply:
Changes made.